### DISCUSSION TITLE
Redactor .js configs

### DISCUSSION THREAD

**Discussion by brandonkelly** [craftcms/cms#8386] [Ideas]
It's not currently possible to set Redactor config settings that accept a function, like the ones [documented here](https://imperavi.com/redactor/docs/callbacks/link/), because you’re limited to things you can define with JSON.

This could be made possible if, in addition to `.json` Redactor configs, the Rich Text field supported `.js` configs, which could execute arbitrary JS code.

**Comment by zauni** [comment]
This would be great for example to have an opportunity to encode email links against spam bots.
